README.md

```diff
@@ -21,7 +21,7 @@ In this project I created a mobile version of a Portfolio integrating some featu
 
 ## Live Demo
 
-Not available yet
+https://nmeziany.github.io/SkeletonMobilePortfolio/
```

Comment:
- [ ] [OPTIONAL] You can make it like this, maybe it's nicer. But it's up to you.

`[Link](https://nmeziany.github.io/SkeletonMobilePortfolio/)`